int64 audio_frame
uint8[] data
---
string speaker_state
